frontend/src/components/Pages/FrontPage/RobotCards/RobotStatusSection.tsx

```diff
@@ -35,7 +36,15 @@ export const RobotStatusSection = () => {
             (robot) =>
                 robot.currentInstallation.installationCode.toLocaleLowerCase() === installationCode.toLocaleLowerCase()
         )
-        .sort((robot, robotToCompareWith) => (robot.status! > robotToCompareWith.status! ? 1 : -1))
+        .sort((robot, robotToCompareWith) =>
+            robot.status! !== robotToCompareWith.status!
+                ? robot.status! > robotToCompareWith.status!
+                    ? 1
+                    : -1
```

Comment:
1 and -1  are kind of magic numbers. Do they represent True / False, could we use that instead?

Reply:
Kind of. Usually sorting is done by returning a - b but when sorting by something that isn't a number this is the common way to do it in typescript.

Reply:
OKey, good clarification, thank you